### DISCUSSION TITLE
Is it possible to use Apache AGE extension?

### DISCUSSION THREAD

**Discussion by moaali** [cloudnative-pg/cloudnative-pg#2122] [Q&A] [answered]
Just a noop question here, is it possible to use [Apache AGE](https://github.com/apache/age) with CloudNativePG? and what about other PG extensions?

**Comment by gbartolini** [comment, marked as answer]
I am not familiar with the AGE extension but, as long as you add it to your container images and you follow the instructions (e.g. create extension in the database, load the library, and so on) it should work.

My advice is to look at how the postgres-containers project is done (you could fork it for example and add the library in the docker file). Also this page is IMHO useful: https://cloudnative-pg.io/documentation/current/container_images/

Let me know if this answers your question.
